tests/test_edgeql_data_migration.py

```diff
@@ -10869,18 +10869,18 @@ async def test_edgeql_migration_eq_collections_25(self):
                     alias Foo := [20];
                 }
             };
-            POPULATE MIGRATION;
-            COMMIT MIGRATION;
         """)
+        await self.con.execute("POPULATE MIGRATION")
```

Comment:
Why are these pulled out?  We ought to support `start/populate/commit` as a single script.  While these are implicit transaction boundaries, they are _not_ the same as explicit `start transaction/commit`.

Reply:
@fmoor I'll amend a fix to allow migration commands in scripts